src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -923,6 +923,11 @@ private function resolveServices(mixed $value, string $file, bool $isParameter =
 
             return new Expression(substr($value, 2));
         } elseif (\is_string($value) && str_starts_with($value, '@')) {
+            if (str_starts_with($value, '@>')) {
+                $argument = $this->resolveServices(str_replace('@>', '@', $value), $file, $isParameter);
```

Comment:
substr instead of str_replace would be more appropriate

Reply:
It's exactly because of @>? the str_replace works better

Reply:
but nothing prevents having a `@>` in the middle of the service id, we don't want to be loose on this